File: Data Structures & Algorithms/missing-number/submission-1.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int sum2=0;

        for(int i=0;i<n;i++){
            sum2=sum2+nums[i];
        }

        return (n+1)*(n)/2 -sum2;
    }
}

File: Data Structures & Algorithms/three-integer-sum/submission-1.py
def verif(lm,pm):
    l=sorted(lm)
    m=sorted(pm)
    if l==m:
        return True
    return False



class Solution:
    def threeSum(self, nums):
        liste=[]
        vu=[]
        for i in range(len(nums)):
            if not i in vu:
                vu.append(i)

                for j in range(len(nums)):
                    a=-(nums[i]+nums[j])
                    if j!=i and a in nums :

                        for k in range(len(nums)):
                            if nums[k]==a and i!=k and j!=k:
                                cop=True
                                lmk=[nums[i],nums[j],nums[k]]
                                for g in liste:
                                    if verif(g,lmk): cop=False
                                
                                if cop :
                                    liste.append(lmk)
        return liste

                            

        